lib/nrf_modem_lib/nrf_modem_os.c

```diff
@@ -343,12 +343,12 @@ void nrf_modem_os_log(int level, const char *fmt, ...)
 	va_list ap;
 	va_start(ap, fmt);
 
-	if (IS_ENABLED(CONFIG_LOG_MODE_MINIMAL)) {
+#if IS_ENABLED(CONFIG_LOG_MODE_MINIMAL)
```

Comment:
`IS_ENABLED()` is meant to be used with the `if` statement; use `#if` here, or `#ifdef`.